src/heretic/g_game.c

```diff
@@ -621,6 +626,34 @@ void G_BuildTiccmd(ticcmd_t *cmd, int maketic)
             BT_SPECIAL | BTS_SAVEGAME | (savegameslot << BTS_SAVESHIFT);
     }
 
+    // low-res turning

```

Comment:
This comment doesn't convey anything that the if() statement below doesn't already convey. Either remove or (better) expand the comment explaining some of the background to the feature and why it's required.